### DISCUSSION TITLE
为什么浏览器有时会打开空白，后台管理也是空白

### DISCUSSION THREAD

**Discussion by qadan72** [AlistGo/alist#420] [Q&A] [answered]
Chrome（97.0.4692.99）我自己网站也是这样，作者站也是这样，火狐浏览器也是打开空白，有两个js是404，不知道是不是这个原因，
js可以本地化吗，不在引用其他链接

![image](https://user-images.githubusercontent.com/40349902/150663882-0cedfa4b-3f61-4e87-a8e9-291b8281ee09.png)

![image](https://user-images.githubusercontent.com/40349902/150663901-917f8d47-ed29-47bd-9347-94bd36d21c0c.png)

**Comment by xhofe** [comment, marked as answer]
- #227
- #411
- https://alist-doc.nn.ci/docs/setting/config#assets
